src/arrayMethodSort.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    a.toString() > b.toString()) {
```

Comment:
The compareFunction should return a number, not a boolean. You should subtract 'b' from 'a' instead of comparing them with '>'.

`String(a) > String(b) ? 1 : -1;`